clusters/build-clusters/01_cluster/ci/_origin-release-build/golang-1.17/Dockerfile

```diff
@@ -0,0 +1,58 @@
+#
+# This is the image that controls the standard build environment for releasing OpenShift Origin.
+# It is responsible for performing a cross platform build of OpenShift.
+#
+# The standard name for this image is openshift/origin-release
+#
+FROM        registry.centos.org/centos/centos:7
```

Comment:
Would it be worth trying out to make this image centos:8 based instead of the 7 one? Or would it require too many changes?

Reply:
I tried that yesterday but lot of dependencies were missing, should be explored as a separate item..

Reply:
I'm not insisting, just wondering. If it woudl need too much work, let's leave it as it is

Reply:
I built this image locally and works fine, wondering if there is a way to build in the CI as a rehearse or someone needs to check manually?

Reply:
Unfortunately not for these: it's unfortunately a pile of unmaintained manifests :( Needs a manual check after submission. But I think it's worth a shot - if you add the changes as a new commit, at worst we can revert that commit and go back to centos7.

Reply:
@petr-muller I have added centos:8 change but I doubt it will work :)